geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionStats.java

```diff
@@ -947,6 +949,7 @@
 
   private final MaxLongGauge maxReplyWaitTime;
   private final MaxLongGauge maxSentMessagesTime;
+  private AtomicInteger serialQueueBytes = new AtomicInteger();
```

Comment:
How about AtomicLong instead? The int stats are now all actually long stats. You might as well make this AtomicLong and change getInternalSerialQueueBytes to return long instead of int.

Reply:
If the serial bytes are being updated from lots of threads and only read occasionally then you should use `LongAdder` which is what stats is using internally.

Reply:
Fixed